school_student.rb

```diff
@@ -0,0 +1,10 @@
+class Student < Person
+  def initialize(classroom)
+    super
+    @classroom = classroom
+  end
```

Comment:
- [ ] [optional] The Student initializer extends the parent's initializer and should also accept parameters that are required for the parent class person. I would suggest considering the below code.
  - `def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
    @classroom = classroom
  end`